plugins/routed-eni/cni.go

```diff
@@ -179,6 +179,25 @@ func add(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 	if err != nil {
 		log.Errorf("Failed SetupPodNetwork for pod %s namespace %s container %s: %v",
 			string(k8sArgs.K8S_POD_NAME), string(k8sArgs.K8S_POD_NAMESPACE), string(k8sArgs.K8S_POD_INFRA_CONTAINER_ID), err)
+
```

Comment:
Do we want metrics for this branch?

Reply:
Yes. there is a metrics for this in ipamd.go, and reason indicates why Pod is deleted.
```
+       addIPCnt = prometheus.NewCounter(
+               prometheus.CounterOpts{
+                       Name: "add_ip_req_count",
+                       Help: "The number of add IP address request",
+               },
+       )
+       delIPCnt = prometheus.NewCounterVec(
+               prometheus.CounterOpts{
+                       Name: "del_ip_req_count",
+                       Help: "The number of delete IP address request",
+               },
+               []string{"reason"},
+       )
```